apps/student/src/components/homepage/SkillsProgress.tsx

```diff
@@ -170,6 +61,8 @@ export const SkillsProgress = () => {
     const color = skillColor ?? theme.palette.primary.main;
     const progress = Math.floor(((nodeIndex + 1) / totalNodes) * 100);
 
+    completed = nodeIndex + 1 !== totalNodes;
```

Comment:
is this ok? will this trigger a re-render in order to see the updated `completed` value?